changelog/22.0/22.0.0/summary.md

```diff
@@ -54,6 +56,16 @@ $ vtctldclient ApplySchema --ddl-strategy="gh-ost" ...
 $ vtctldclient ApplySchema --ddl-strategy="pt-osc" ...
 ```
 
+### <a id="vtgate-metrics"/>VTGate Metrics
+
+Added two new metrics for queries with query type, plan type and tablet type as dimension.
+1. QueryProcessed - This counts the number of query executed.
+2. QueryRouted - This counts the number of shards the query was executed on.
+
+Deprecated:
+1. QueriesProcessed
```

Comment:
In code you are deprecating the table metrics, but that is missing from here.

Reply:
I have added `QueryTables` as replacement metric and updated the summary notes.